modules/network/src/main/java/org/apache/ignite/network/internal/netty/ConnectionManager.java

```diff
@@ -116,11 +117,17 @@ public ConnectionManager(
      */
     public void start() throws IgniteInternalException {
         try {
-            server.start().join();
+            server.start().get(3, TimeUnit.SECONDS);
```

Comment:
We don't have any failover timeout in network configuration right now. Please link this part with TODO to some ticket that we'll use to create such configuration.

Reply:
sorry, I don't understand what are you trying to say. Do you mean that we should a common configuration for all timeouts in the module?

Reply:
I mean this this should not be hardcoded

Reply:
added a TODO